frontend/src/services/EnvService.ts

```diff
@@ -0,0 +1,23 @@
+import { UiCookieValues } from '~shared/types'
+
+import { PUBLIC_FORMS_ENDPOINT } from '~features/public-form/PublicFormService'
+
+import { ApiService } from './ApiService'
+
+const ENV_ENDPOINT = '/environment'
+const ADMIN_ENDPOINT = '/admin'
+export const publicChooseEnvironment = async (
+  ui: UiCookieValues,
+): Promise<UiCookieValues> => {
+  return ApiService.get(`${PUBLIC_FORMS_ENDPOINT}/${ENV_ENDPOINT}/${ui}`).then(
```

Comment:
instead of allowing a param, could we not hardcode the `ui` param in this function instead? since this is the react directory and we are only allowing a switch to AngularJS variant, haha

Reply:
ok makes sense! will hardcode it